rpcs3/rpcs3qt/gui_settings.h

```diff
@@ -190,7 +190,8 @@ namespace gui
 
 	const gui_save l_tty   = gui_save(logger, "TTY",   true);
 	const gui_save l_level = gui_save(logger, "level", (uint)(logs::level::success));
-	const gui_save l_stack = gui_save(logger, "stack", true);
+	const gui_save l_stack = gui_save(logger, "log stack", true);
```

Comment:
I meant the string. Not the variable

Reply:
Trying to maintain clarity between stacking the log and stacking the tty.

Reply:
It's already in the log section in a file noone should exit by hand. What else would the stack be for?

Reply:
Sorry, got it confused with the label for the checkbox.